pkg/config/loader.go

```diff
@@ -73,13 +94,14 @@ func (l *Loader) handleGoVersion() {
 
 	l.cfg.LintersSettings.ParallelTest.Go = l.cfg.Run.Go
 
-	trimmedGoVersion := trimGoVersion(l.cfg.Run.Go)
-
-	l.cfg.LintersSettings.Gocritic.Go = trimmedGoVersion
 	if l.cfg.LintersSettings.Gofumpt.LangVersion == "" {
 		l.cfg.LintersSettings.Gofumpt.LangVersion = l.cfg.Run.Go
 	}
 
+	trimmedGoVersion := trimGoVersion(l.cfg.Run.Go)
+
+	l.cfg.LintersSettings.Gocritic.Go = trimmedGoVersion
+
```

Comment:
Could you please explain the reason for moving these lines?

Reply:
Because the empty line was removed in a previous PR by error.
I just separate each linter.